action.yml

```diff
@@ -121,6 +125,21 @@ runs:
             TWINE_OPTS="$TWINE_OPTS --skip-existing"
           fi
           python -m pip install twine
+          if [[ -z "$TWINE_PASSWORD" ]]; then
+            # vis https://docs.pypi.org/trusted-publishers/using-a-publisher/#the-manual-way
+            export TWINE_PASSWORD=$(
+        python -c 'import json, urllib.request as req
+        audience = req.urlopen("'${INPUT_REGISTRY_DOMAIN}'/_/oidc/audience").read()
+        token = json.load(req.urlopen(req.Request(
+            url="'${ACTIONS_ID_TOKEN_REQUEST_URL}'/_/oidc/audience",
```

Comment:
The `/…/audience` suffix shall be deleted.

Reply:
Future tense? As in this is fine as-is for now?